.github/workflows/tests.yml

```diff
@@ -0,0 +1,21 @@
+name: Tests
+
+on:
+  push:
+    branches: ["develop", "release-*"]
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9"]
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - run: pip install tox
+      - run: tox -e tests
```

Comment:
There are probably some shenanigans you can pull to avoid having to specify `-e tests` here (keeping it all defined in `tox.ini`'s default testenv), since I _think_ the convention for Tox is to default to running unit tests, but again, totally optional / up to you.